codegen-renderers/src/test/kotlin/io/vrap/codegen/languages/TestCodeGenerator.kt

```diff
@@ -42,6 +45,38 @@ class TestCodeGenerator {
         generatorComponent.generateFiles()
     }
 
+    @Test
+    fun generateJavaModelsWithInterfacesModule() {
+        cleanGenTestFolder()
```

Comment:
I think this is unnecessary since the output folder is cleaned up automatically.